src/presets/firebase.ts

```diff
@@ -68,7 +68,10 @@ async function writeRoutes(nitro: Nitro) {
   const _require = createRequire(import.meta.url);
 
   const jsons = await globby(
-    join(nitro.options.output.serverDir, "node_modules/**/package.json")
+    join(
+      nitro.options.output.serverDir,
+      "node_modules/**[!fsevents]/package.json"
+    )
```

Comment:
I suggest this version which works and uses the native filtering feature of globby

```suggestion
    join(nitro.options.output.serverDir, 'node_modules/**/package.json'),
    // Avoid packages that are specific to some platforms and create
    // un-deployable builds
    // https://github.com/unjs/nitro/issues/1367
    { ignore: ['**/fsevents/**'] }
```